CrossSIMD.h:673:33: error: cannot convert 'float32x4_t' to 'int32x4_t'

### Platform

libretro / Retroarch

### Compiler and build tool versions

CMAKE 3.24.2

### Operating system version

Linux 4.9 Emuelec

### Build commands used

PROJECT=Amlogic-ce DEVICE=Amlogic-ng ARCH=aarch64 DISTRO=EmuELEC ./scripts/build ppsspp

### What happens

Failed build due to cannot convert 'float32x4_t' to 'int32x4_t'

logs

In file included from /home/kelv/clone3/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-d7aa236e1343956b614e75771f1c8b35eb0aa96d/Core/HLE/HLE.cpp:23:
build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-d7aa236e1343956b614e75771f1c8b35eb0aa96d/Common/Math/CrossSIMD.h: In function 'bool AnyZeroSignBit(Vec4F32)':
build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-d7aa236e1343956b614e75771f1c8b35eb0aa96d/Common/Math/CrossSIMD.h:673:33: error: cannot convert 'float32x4_t' to 'int32x4_t'
  673 |         return vmaxvq_s32(value.v) >= 0;
      |                           ~~~~~~^

### PPSSPP version affected

https://github.com/hrydgard/ppsspp/commit/a5116e15907ce6b2b05f7b5b330e9fac4cecd0bd

### Last working version

https://github.com/hrydgard/ppsspp/commit/a40cf125934847b6ba09cc87ea9f2c4194ddb378

### Checklist

- [x] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [x] Make sure to run `git submodule update --init --recursive` before building.
- [x] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [x] Try deleting the build directory and running the build again.
- [x] Check GitHub Actions, which builds every merge and PR.
- [x] Include logs and help us reproduce.

--- a/Common/Math/CrossSIMD.h
+++ b/Common/Math/CrossSIMD.h
@@ -670,7 +670,7 @@ inline bool AnyZeroSignBit(Vec4F32 value) {
 	int32x4_t ival = vreinterpretq_s32_f32(value.v);
 #if PPSSPP_ARCH(ARM64_NEON)
 	// Shortcut on arm64
-	return vmaxvq_s32(value.v) >= 0;
+	return vmaxvq_s32(ival) >= 0;
 #else
 	int32x2_t prod = vand_s32(vget_low_s32(ival), vget_high_s32(ival));
 	int mask = vget_lane_s32(prod, 0) & vget_lane_s32(prod, 1);

